outputs.tf

```
@@ -29,6 +29,16 @@ output "eks_node_group_status" {
 }
 
 output "eks_node_group_remote_access_security_group_id" {
-  description = "The ID of the security group generated to allow SSH access to the nodes, if this module generated one"
-  value       = join("", aws_security_group.remote_access.*.id)
+  description = "ID of the EKS cluster Security Group for remote access to EKS Node Group"
+  value       = module.security_group.id
+}
+
+output "eks_node_group_remote_access_security_group_arn" {
+  description = "ARN of the EKS cluster Security Group for remote access to EKS Node Group"
+  value       = module.security_group.arn
+}
+
+output "eks_node_group_remote_access_security_group_name" {
+  description = "Name of the EKS cluster Security Group for remote access to EKS Node Group"
+  value       = module.security_group.name
 }
```

Also (or in place of additional outputs) output the entire `module.security_group`

```suggestion
}

output "eks_node_group_remote_access_security_group" {
  description = "EKS cluster Security Group for remote access to EKS Node Group"
  value       = module.security_group
}
```